gradle/verification-metadata.xml

```diff
@@ -1515,6 +1522,11 @@
             <pgp value="8B16E798477BE6DC10127D11AA6917FA45E67F20"/>
          </artifact>
       </component>
+      <component group="org.testcontainers" name="testcontainers" version="1.16.2">
```

Comment:
🤔 usually indirectly forced to `1.12.5` by `org.testcontainers:spock` should we update that as well so only one version gets resolved throughout the build?

Reply:
See #27793